### ISSUE TITLE
Can't debug stored procedures.

### ISSUE THREAD

**Issue by ukolovda** [ng-galien/idea-plpgdebugger#21]
**Describe the bug**
Can't debug stored procedures.

**To Reproduce**
```sql
create or replace procedure debug_proc_test() as $$
begin
  null;
end;
$$ language plpgsql;

-- Select here and click "Debug" button:
call debug_proc_test();
```

**Expected behavior**
Should debug procedures (not functions only),

**Screenshots**
![изображение](https://user-images.githubusercontent.com/1921594/150672518-e97c33b1-1f04-4fae-96e1-2e26c03a85ca.png)

**Comment by ng-galien** [comment]
Hi, the query used for detection is [here](https://github.com/ng-galien/idea-plpgdebugger/blob/main/src/main/kotlin/net/plpgsql/ideadebugger/SQLQuery.kt#L135) for further investigation

**Comment by ng-galien** [comment]
@ukolovda Will be available in the next release

**Comment by ng-galien** [comment]
Fixed in 1.0.2

**Comment by nvs2022** [comment]
I have same case, but nothing happens
if replace procedure -> function, all working!
